src/test/java/com/iota/iri/TransactionValidatorTest.java

```diff
@@ -1,58 +1,68 @@
 package com.iota.iri;
 
-import com.iota.iri.conf.MainnetConfig;
 import com.iota.iri.controllers.TipsViewModel;
 import com.iota.iri.controllers.TransactionViewModel;
 import com.iota.iri.crypto.SpongeFactory;
 import com.iota.iri.model.TransactionHash;
 import com.iota.iri.network.TransactionRequester;
 import com.iota.iri.service.snapshot.SnapshotProvider;
-import com.iota.iri.service.snapshot.impl.SnapshotProviderImpl;
+import com.iota.iri.service.snapshot.impl.SnapshotMockUtils;
 import com.iota.iri.storage.Tangle;
 import com.iota.iri.storage.rocksDB.RocksDBPersistenceProvider;
 import com.iota.iri.utils.Converter;
 import org.junit.AfterClass;
+import org.junit.Before;
 import org.junit.BeforeClass;
 import org.junit.Test;
 import org.junit.rules.TemporaryFolder;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnitRunner;
 
 import static com.iota.iri.TransactionTestUtils.*;
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertTrue;
 
+@RunWith(MockitoJUnitRunner.class)
```

Comment:
Can you replace this by adding:
```
  @Rule
    public MockitoRule mockitoRule = MockitoJUnit.rule();
```

The reason is this is the way we currently do it in the project.
I personally think that `@RunWith(MockitoJUnitRunner.class)` is nicer  :-)
But, we don't want to change it in all the unit tests in the project

Reply:
no problem. I’ll switch it.

Reply:
are you doing it?

Reply:
Yep, finally found some time to get to it. Changed in all places I added the runner before.

Reply:
I thought you were going to resolve conflicts as well.
Do you intend to?

Reply:
You are right, just did it. Fixed conflicts and tests still passing.